src/components/SøknadsSteg/Steg/Steg.tsx

```diff
@@ -3,20 +3,24 @@ import './Steg.less';
 import KnappBase from 'nav-frontend-knapper';
 import Stegindikator from 'nav-frontend-stegindikator';
 import Panel from 'nav-frontend-paneler';
-import { StegRoutes } from '../../../routing/Routes';
+import { StegRoutes, RouteEnum } from '../../../routing/Routes';
 import { Systemtittel, Ingress } from 'nav-frontend-typografi';
 import { useLocation, useHistory } from 'react-router-dom';
 import { IStegRoute, hentNesteRoute, hentForrigeRoute } from '../../../routing/Routes';
+import { Hovedknapp } from 'nav-frontend-knapper';
+import { hentPath } from '../../../routing/Routes';
+import { ILokasjon } from '../../../typer/lokasjon';
 
 interface ISteg {
     tittel: string;
+    erSpørsmålBesvart: boolean;
 }
 
-const Steg: React.FC<ISteg> = ({ tittel, children }) => {
-    const location = useLocation();
+const Steg: React.FC<ISteg> = ({ tittel, children, erSpørsmålBesvart }) => {
```

Comment:
Kan erSpørsmålBesvart settes til true ved default?

Reply:
kan helt sikkert det akkurat nå, men vet ikke om det er noe vits for alle sider skal vel sende inn en condition for om alt på siden er besvart

Reply:
sant